### PR TITLE
Add/Update Documentation for Media Library Config

### PR DESCRIPTION
**Pull request by DaRaFF**

### CODE REVIEW

File: content/enterprise/reference-docs/server-configuration/_index.md [hunk 1]
```diff
@@ -741,28 +620,125 @@ The asset management impacts editor in four ways:
 {{< added-in release-2021-03 >}}
 - Customize indexing of metadata is supported
 
-##### Prerequisite
+{{< added-in release-2021-06 >}}
+- Files are supported
```
Comment (mokimo) [inline content/enterprise/reference-docs/server-configuration/_index.md:624]:
```suggestion
- Files are supported (for example PDFs)
```

File: content/enterprise/reference-docs/server-configuration/_index.md [hunk 2]
```diff
@@ -741,28 +620,125 @@ The asset management impacts editor in four ways:
...
+- The config `mediaLibrary.images` `mediaLibrary.videos` `mediaLibrary.files` is now supported. Before the asset configs were stored in `images`, `videos` and `files`.
```
Comment (mokimo) [inline content/enterprise/reference-docs/server-configuration/_index.md:625]:
```suggestion
- We now support the following configs: `mediaLibrary.images` `mediaLibrary.videos` `mediaLibrary.files` is now supported.
_Before the release-2021-06 - those configs were stored on the top-level 'images', 'videos', 'files'_
```

File: content/enterprise/reference-docs/server-configuration/_index.md [hunk 3]
```diff
@@ -741,28 +620,125 @@ The asset management impacts editor in four ways:
...
+      maxFileSize: 15 * 1000 * 1000 // 15MB, defaults to 15MB.
     }
   }
 }
 ```
-[config options]({{< ref "/enterprise/reference-docs/server-configuration/storage" >}}) for `storage`.
 
 
-##### Setting up the media-library Elastic Search Mapping
+##### Setting up the MediaLibrary Elastic Search Mapping
```
Comment (mokimo) [inline content/enterprise/reference-docs/server-configuration/_index.md:741]:
```suggestion
##### Setting up the Media Library Elastic Search Mapping
```

File: content/enterprise/reference-docs/server-configuration/_index.md [hunk 4]
```diff
@@ -780,14 +756,14 @@ The name of the Media index has to be configured:
...
-npx livingdocs-server es-media-reindex -y
+```js
+livingdocs-server elasticsearch-index --handle li-media -y // ({{< added-in release-2021-03 >}})
 ```
 
 To index the metadata the plugin must support the mediaIndex and on the metadata in the mediaType the index must be enabled.
 
 ###### Enable mediaIndex on metadata plugin
-To enable the indexing for the media-library in a plugin you have to add the `mediaIndex` config.
+To enable the indexing for the MediaLibrary in a plugin you have to add the `mediaIndex` config.
```
Comment (mokimo) [inline content/enterprise/reference-docs/server-configuration/_index.md:766]:
```suggestion
To enable the indexing for the Media Library in a plugin you have to add the `mediaIndex` config.
```

**Review by mokimo — APPROVED** [review]

**Comment by mokimo** [comment]
Haven't checked out all the configs in detail or if we still point to 'images' 'videos' 'files' somewhere else, but I'll trust that you looked at that thoroughly 👍

**Comment by DaRaFF** [comment]
@mokimo never trust me :-)

**Pull request merged**